indigo_app/forms/tasks.py

```diff
@@ -196,7 +196,7 @@ def __init__(self, country, *args, **kwargs):
     def clean_assigned_to(self):
         user = self.cleaned_data['assigned_to']
         if user and self.country not in user.editor.permitted_countries.all():
-            raise forms.ValidationError("That user doesn't have appropriate permissions for {}".format(self.country.name))
+            raise forms.ValidationError(_("That user doesn't have appropriate permissions for {}").format(self.country.name))
```

Comment:
My understanding is that you need to used named parameter interpolation with translations: https://docs.djangoproject.com/en/5.0/topics/i18n/translation/#standard-translation

So `%(country)s` etc.

Reply:
@longhotsummer updated